dev/com.ibm.ws.jaxrs.2.x.monitor/src/com/ibm/websphere/jaxrs/monitor/RestStatsMXBean.java

```diff
@@ -12,63 +12,131 @@
-
-    /**
-     * Retrieves the value of the read-only attribute RequestCount, the number of requests the server has received for this resource method.
-     * 
-     * @return request count
-     */
-    public long getRequestCount();
-
-    /**
-     * Retrieves the value of the read-only attribute RequestCountDetails, which provides other details on the request count.
-     * 
-     * @return request count details
-     */
-    public Counter getRequestCountDetails();
-
-    /**
-     * Retrieves the value of the read-only attribute ResponseTime, which is the average (mean) time spent responding to each request for the resource method.
-     * 
-     * @return response time
-     */
-    public double getResponseTime();
-
-    /**
-     * Retrieves the value of the read-only attribute ResponseCountDetails, which provides statistical details on the response time.
-     * 
-     * @return response time details
-     */
-    public StatisticsMeter getResponseTimeDetails();
-
-    /**
-     * Retrieves the value of the read-only attribute AppName, the name of the application of which the resource method belongs.
-     * 
-     * @return app name
-     */
-    public String getAppName();
+	/**
+	 * Retrieves the value of the read-only attribute Description, which is a
+	 * description of the MBean itself.
+	 * 
+	 * @return description
+	 */
+	public String getDescription();
+
+	/**
+	 * Retrieves the value of the read-only attribute MethodName, the name of the
+	 * resource method as specified in the deployment descriptor.
+	 * 
+	 * @return method name
+	 */
+	public String getMethodName();
+
+	/**
+	 * Retrieves the value of the read-only attribute RequestCount, the number of
+	 * requests the server has received for this resource method.
+	 * 
+	 * @return request count
+	 */
+	public long getRequestCount();
+
+	/**
+	 * Retrieves the value of the read-only attribute RequestCountDetails, which
+	 * provides other details on the request count.
+	 * 
+	 * @return request count details
+	 */
+	public Counter getRequestCountDetails();
+
+	/**
+	 * Retrieves the value of the read-only attribute ResponseTime, which is the
+	 * average (mean) time spent responding to each request for the resource method.
+	 * 
+	 * @return response time
+	 */
+	public double getResponseTime();
+
+	/**
+	 * Retrieves the value of the read-only attribute ResponseCountDetails, which
+	 * provides statistical details on the response time.
+	 * 
+	 * @return response time details
+	 */
+	public StatisticsMeter getResponseTimeDetails();
+
+	/**
+	 * Retrieves the value of the read-only attribute MinuteLatestMinimumDuration,
+	 * which provides details of the minimum duration of the latest, most-recently,
+	 * recorded complete minute (latest minute can be on-going and not "complete" 
+	 * if mbean is being updated in the current minute).
+	 * 
+	 * @return minimum elapsed duration of latest minute
+	 */
+	public Gauge getMinuteLatestMinimumDuration();
```

Comment:
These are API breaking changes!

Reply:
Currently not exactly a public API. This is addressed in https://github.com/OpenLiberty/open-liberty/pull/12530.

Thus we can modify the API at the moment.

Further discussion shows that adding methods to the API on necessitates a package-info version change.

The modifications here will be considered part of the 1.0 API.

Reply:
I don't see the package-info.java got updated here.